### PR TITLE
Accept 5D weights in VolumetricConvolutionMM

### PR DESCRIPTION
**Pull request by apaszke**

### CODE REVIEW

File: lib/THNN/generic/VolumetricConvolutionMM.c [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
 
+static int THNN_(view_weight)(THTensor **_weight)
+{
+  THTensor *weight = *_weight;
+  THArgCheck(weight->nDimension == 2 || weight->nDimension == 5, 4,
+          "weight tensor should be 2D or 5D - got %dD", weight->nDimension);
+  if (weight->nDimension == 5) {
+    long s1 = weight->size[0];
+    long s2 = weight->size[1] * weight->size[2] * weight->size[3] * weight->size[4];
+    *_weight = THTensor_(newWithStorage2d)(weight->storage, 0, s1, -1, s2, -1);
```
Comment (fmassa) [inline lib/THNN/generic/VolumetricConvolutionMM.c]: Need to add `storageOffset` here, as was included in
https://github.com/torch/nn/blob/master/lib/THNN/generic/SpatialConvolutionMM.c#L104

Comment (soumith) [inline lib/THNN/generic/VolumetricConvolutionMM.c]: @apaszke same mistake again :P

Comment (soumith) [inline lib/THNN/generic/VolumetricConvolutionMM.c]: done via https://github.com/torch/nn/commit/8f2b88a5a4f2b541b9efc2fb72a47572811efd17

Comment (apaszke) [inline lib/THNN/generic/VolumetricConvolutionMM.c]: Damn, I must have copied a snipped from the old commit 😕 Sorry for that.

**Pull request merged**